Makefile

```diff
@@ -4,14 +4,15 @@ IPXE_BUILD_SCRIPT := binary/script/build_ipxe.sh
 IPXE_FETCH_SCRIPT := binary/script/fetch_and_extract_ipxe.sh
 IPXE_NIX_SHELL := binary/script/shell.nix
 IPXE_ISO_BUILD_PATCH := binary/script/iso.patch
+BINARIES := binary/ipxe.efi binary/snp.efi binary/undionly.kpxe binary/ipxe.iso binary/ipxe-efi.img
 
 help: ## show this help message
 	@grep -E '^[a-zA-Z_-]+.*:.*?## .*$$' Makefile | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[32m%-30s\033[0m %s\n", $$1, $$2}'
 
 include lint.mk
 
 .PHONY: binary
-binary: binary/ipxe.efi binary/snp.efi binary/undionly.kpxe binary/ipxe.iso ## build all upstream ipxe binaries
+binary: $(BINARIES)
```

Comment:
ah sorry, not blocking on this PR, but mind adding back the comment? it gets parsed for the help target. `## build all upstream ipxe binaries`

Reply:
Will fix.

Reply:
Now fixed in the CORRECT COMMIT!